domain/model/bootstrap/bootstrap.go

```diff
@@ -70,12 +89,25 @@ func CreateModel(
 				return fmt.Errorf("determining cloud type for model %q: %w", args.Name, err)
 			}
 
-			if err := state.Create(ctx, tx, modelType, args); err != nil {
-				return fmt.Errorf("create bootstrap model %q with uuid %q: %w", args.Name, args.UUID, err)
+			if args.SecretBackend == "" && modelType == coremodel.CAAS {
+				args.SecretBackend = kubernetessecrets.BackendName
+			} else if args.SecretBackend == "" && modelType == coremodel.IAAS {
+				args.SecretBackend = jujusecrets.BackendName
```

Comment:
OT: Nothing you've done, but those backend names should be centralised.

Reply:
@wallyworld this is one that you need to consider in your work please.